main/pipeline/pipeline.cc

```diff
@@ -157,7 +157,7 @@ core::LocOffsets locOffset(pm_location_t *loc, pm_parser_t *parser) {
     return core::LocOffsets{locStart, locEnd};
 }
 
-const unique_ptr<parser::Node> convertPrismToSorbet(pm_node_t *node, pm_parser_t *parser, core::GlobalState &gs) {
+unique_ptr<parser::Node> convertPrismToSorbet(pm_node_t *node, pm_parser_t *parser, core::GlobalState &gs) {
```

Comment:
I have a C++ question here -- the reason I removed this `const` is because I was having trouble [reassigning `ifTrue` on line 185 below](https://github.com/Shopify/sorbet/pull/31/commits/867ee7d1bd0d1a711c429291c984172f2df10710#diff-9649cbc5bd643f35c408d9a4e882a42b40637df8574fe71ca2e5df8bcfc1d2b6R185). Is there a better way to write the code below such that I wouldn't have to remove `const` here?

Reply:
Does it error because of the `std::move(ifTrue)` later?